book/src/css.md

```diff
@@ -0,0 +1,264 @@
+# CSS
+
+When you want to modify the style of your website, you use [CSS](https://de.wikipedia.org/wiki/Cascading_Style_Sheets).
+Since so many people are already familiar with web development, it makes sense that GTK supports its own variant of CSS.
+
+> We will not explain every piece of CSS syntax used in this chapter.
+> If you are new to it or just need a refresher, have a look at the [MDN Web Docs](https://developer.mozilla.org/en-US/docs/Web/CSS/Syntax)
+
+Let us say we have a button and we want to set its font color to red.
+Every widget has a corresponding CSS node.
+In the case of `gtk::Button`, this node is called `button`.
+Therefore, we create a `style.css` file with the following content:
+
+<span class="filename">Filename: listings/css/1/style.css</span>
+
+```css
+{{#rustdoc_include ../listings/css/1/style.css}}
+```
+
+Next, we load the CSS file in the startup step of the application.
```

Comment:
Might be a good idea to add a small section about testing the CSS in the inspector. This way you can make sure it works and then when you're loading it into the app you know that if it doesn't work then it's the loading code and not the CSS.

Reply:
So basically move the inspector blue message from below here and reword it a bit.